data/json/items/book/misc.json

```
@@ -1067,47 +1027,45 @@
     "type": "BOOK",
     "name": "sports novel",
     "description": "The dramatic tale of a small-time boxer who gets a rare chance to fight the heavy-weight champion, and seize his one chance to make a better life for himself while impressing the cute girl who works in the pet store.",
-    "weight": "688 g",
-    "volume": "750 ml",
-    "price": 750,
-    "price_postapoc": 1900,
-    "material": [ "paper" ],
-    "symbol": "?",
-    "color": "light_blue",
+    "copy-from": "paperback_novel",
     "intelligence": 7,
     "time": "20 m",
     "chapters": 28,
-    "fun": 4
+    "fun": 3
   },
   {
     "id": "novel_spy",
     "type": "BOOK",
-    "name": "spy novel",
+    "name": { "str": "spy novel", "str_pl": "spy novels" },
```

```suggestion
    "name": "spy novel",
```
You don't need to specify "str_pl" if it's simply "str" plus an 's'.